Src/main.c

```diff
@@ -49,7 +49,7 @@
 /* USER CODE END Includes */
 
 /* Private variables ---------------------------------------------------------*/
-
+ADC_HandleTypeDef hadc1;
```

Comment:
Take a look in `Inc/adc.h` and `Src/adc.h` and you will notice that an `ADC_HandleTypeDef` already exists and is initialized in `MX_ADC_Init()`.